06week/README.md

```diff
@@ -0,0 +1,5 @@
+Created an array of people to choose from of eligible players, with different attributes for each of them.
+Created different classes to hold the attributes and move them from array to array, and display them in the DOM. Created a listPeopleChoices function to create new players and map through the array of people. Creates a button that allows you to make a player that when clicked moves the object to the listOfPlayers array. Once listOfPlayers is created they are given two button options for the red or blue team which when clicked will move them to the selected array.
+step 1: Array of People
+step 2: ARRAY OF PLAYERS
+STEP 3: array of red or blue team
```

Comment:
One of the requirements was a test plan write up, or some tests. I dont see either of those.